packages/headless/src/features/facets/category-facet-set/interfaces/response.ts

```diff
@@ -1,10 +1,4 @@
-import {BaseFacetValue, BaseFacetResponse} from '../../facet-api/response';
-
-export interface CategoryFacetValue extends BaseFacetValue {
-  value: string;
-  path: string[];
-  children: CategoryFacetValue[];
-  moreValuesAvailable?: boolean;
-}
```

Comment:
Replaced by the controller interface.

Reply:
Could we invert the order perhaps? Keeping this interface, but writing everything instead of inheriting, adding documentation here, and then exporting and reusing everywhere else?